examples/tomo/checks/check_axes_cone2d_fp.py

```diff
@@ -0,0 +1,96 @@
+"""Cone beam 2D example for checking that orientations are handled correctly.
+
+Due to differing axis conventions between ODL and the ray transform
+back-ends, a check is needed to confirm that the translation steps are
+done correctly.
+
+Both pairs of plots of ODL projections and NumPy axis sums should look
+similar in the sense that they should show the same features in the
+right arrangement (not flipped, rotated, etc.), up to differences in
+magnification and other cone effects.
+
+This example is best run in Spyder section-by-section (CTRL-Enter).
+"""
+
+# %% Set up the things that never change
+
+import matplotlib.pyplot as plt
+import numpy as np
+import odl
+
+# Set back-end here
+impl = 'astra_cuda'
+# Set a volume shift. This should move the projections in the same direction,
+# and shifts along the projection axis should only change cone effects.
+shift = (0, 25)
+
+img_shape = (100, 150)
+img_max_pt = np.array(img_shape, dtype=float) / 2
+img_min_pt = -img_max_pt
+reco_space = odl.uniform_discr(img_min_pt + shift, img_max_pt + shift,
+                               img_shape, dtype='float32')
+phantom = odl.phantom.indicate_proj_axis(reco_space)
+
+assert np.allclose(reco_space.cell_sides, 1)
+
+# Check projections at 0, 90, 180 and 270 degrees
+grid = odl.RectGrid([0, np.pi / 2, np.pi, 3 * np.pi / 2])
+angle_partition = odl.uniform_partition_fromgrid(grid)
+
+# Make detector large enough to cover the object
+src_radius = 500
+det_radius = 1000
+fan_angle = np.arctan(img_max_pt[1] / src_radius)
+det_size = np.floor(2 * (src_radius + det_radius) * np.sin(fan_angle))
+det_shape = int(det_size)
+det_max_pt = det_size / 2
+det_min_pt = -det_max_pt
+detector_partition = odl.uniform_partition(det_min_pt, det_max_pt, det_shape)
+
+assert np.allclose(detector_partition.cell_sides, 1)
+
+# Sum manually using Numpy
+sum_along_x = np.sum(phantom.asarray(), axis=0)
```

Comment:
~~I think if we don't it will try to call `__array_wrap__` and fail.~~ Apparently the array won't be wrapped, so `asarray()` is not necessary.